src/Compilers/Core/Portable/StrongName/StrongNameKeys.cs

```diff
@@ -95,7 +95,6 @@ internal static StrongNameKeys Create(string keyFilePath, CommonMessageProvider
 
             try
             {
-                Debug.Assert(PathUtilities.IsAbsolute(keyFilePath));
```

Comment:
This looks suspicious, we probably shouldn't rely on the current directory and all paths should probably be converted to an absolute path. It is possible that the assert is catching a real issue. CC @tmat